### DISCUSSION TITLE
RegEx for Folder Names

### DISCUSSION THREAD

**Discussion by BAbdulBaki** [WinMerge/winmerge#1552] [Ideas]
I have an online directory structure consisting, for all intents and purposes, of the following structure (space or hyphens for both sides):

Book 1
  Book 1.1
  Book 1.2
Book 2
  Book 2.1
Book 3
etc.

Locally, our folder structure also contains the name of the book after the numbers, as such:

Book 1 ActualName1
  Book 1.1 ActualName1.1
  Book 1.2 ActualName1.2
Book 2 ActualName2
  Book 2.1 ActualName2.1
Book 3 ActualName3
etc.

Is it possible to use RegEx to intelligently map one to the other so that they don't appear as left-only/right-only?

Thanks,
Bassam

**Comment by sdottaka** [comment]
Unfortunately, WinMerge has no such facility for mapping filenames.
